music21/corpus/__init__.py

```diff
@@ -53,13 +53,8 @@
 
 ]
 
-import re
-import os
-import unittest
```

Comment:
really shocked that this didn't cause a pylint error...

Reply:
We have it turned off because we have wildcard imports. And speaking of which, that must have been how identified these, I was looking at pylint reports when poking at #881 (which BTW I think I have a solution for)